src/core/lib/attribute-manager.js

```diff
@@ -513,16 +520,30 @@ export default class AttributeManager {
   // Update attribute buffers from any attributes in props
   // Detach any previously set buffers, marking all
   // Attributes for auto allocation
-  /* eslint-disable max-statements */
+  /* eslint-disable max-statements, no-continue */
   _setExternalBuffers(bufferMap) {
     const {attributes, numInstances} = this;
 
     // Copy the refs of any supplied buffers in the props
     for (const attributeName in attributes) {
       const attribute = attributes[attributeName];
       const buffer = bufferMap[attributeName];
-      attribute.isExternalBuffer = false;
-      if (buffer) {
+
+      if (!buffer) {
+        attribute.isExternalBuffer = false;
+        continue;
```

Comment:
Nit: The rest of the code base avoids continues. Pretty sure this is the only example in about 40K lines.